.travis.yml

```diff
@@ -15,11 +15,14 @@ script:
   - npm run lintNoFix
   - npm run test
 
-  # network simulation tests
+  # network simulation integration tests
   - eval "$(GIMME_GO_VERSION=1.12 gimme)"
   - go get -u github.com/golang/dep/cmd/dep
   - go get -u github.com/btcsuite/btcd
   - npm install
-  - npm run compile
   - npm run test:sim
+  - cat test/simulation/xuddatadir-Alice/logs/xud.log
+  - cat test/simulation/xuddatadir-Bob/logs/xud.log
+  - cat test/simulation/xuddatadir-Carol/logs/xud.log
+  - cat test/simulation/xuddatadir-Dave/logs/xud.log
```

Comment:
Do we want to leave all these statements here? Seems like it would be a lot of output.

Reply:
No, I was adding this to debug the certificate generation issue (`travis debug` commits). Will remove it before merging.